planetiler-custommap/src/main/resources/samples/shortbread.yml

```
@@ -644,6 +644,28 @@ layers:
     - key: kind
       type: match_value
 
+- id: ferries
+  features:
+  - source: osm
+    geometry: line
+    min_zoom:
+      default_value: 12
+      overrides:
+        10:
+          __not__:
+            motor_vehicle: no
+    include_when:
+      route: ferry
+    attributes:
+    - key: kind
+      type: match_value
+    - key: name
+    - key: name_en
+      tag_value: name:en
+    - key: name_de
+      tag_value: name:de
```

I don't have any alias usages left (already at 50), but If I could, this would look like

```suggestion
    - *name
    - *name_en
    - *name_de
```

---

Merge in #1224 then apply this suggestion?